### PR TITLE
Consolidate tfdv.GenerateStats

### PR DESCRIPTION
**Pull request by daikeshi**
A side note:
it probably makes sense now we include `tensorflow-data-validation` in the requirements.txt, since with implementing tfdv workflow, we'll always need tfdv installed. 

@brianmartin @rculbertson

### CODE REVIEW

File: spotify_tensorflow/tfx/tfdv.py [hunk 1]
```diff
@@ -16,25 +16,62 @@
...
+from spotify_tensorflow.tfx.utils import create_setup_file, assert_not_empty_string
+from tensorflow_metadata.proto.v0 import statistics_pb2  # noqa: F401
 
 
-class TFDV(object):
+class GenerateStats(object):
+    def __init__(self, input_data, output_path=None):
+        self.input_data = input_data
+        if output_path is None:
+            self.output_path = os.path.join(self.input_data, "_stats.pb")
```
Comment (andrewsmartin) [inline spotify_tensorflow/tfx/tfdv.py]: (minor) - could we rename `_stats.pb` to `stats.pb`?

Comment (daikeshi) [reply spotify_tensorflow/tfx/tfdv.py]: sure! probably we also need the similar change in mlpr.

File: spotify_tensorflow/tfx/utils.py [hunk 2]
```diff
@@ -30,3 +33,21 @@ def assert_not_empty_string(arg):
...
+def create_setup_file():
+    contents_for_setup_file = """
+    import setuptools
+    
+    if __name__ == "__main__":
+        setuptools.setup(
+            name="spotify_tensorflow_dataflow",
+            packages=setuptools.find_packages(),
+            install_requires=[
+                "spotify-tensorflow[tfdv]"
```
Comment (brianmartin) [inline spotify_tensorflow/tfx/utils.py]: Is this a bug?

Comment (brianmartin) [reply spotify_tensorflow/tfx/utils.py]: Ah, never mind. Is fixed on master.

**Pull request merged**